<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RSVTool</class>
 <widget class="QDialog" name="RSVTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RSV Manager</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>108</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Importing Video Frames</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Exporting Video Frames</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QPushButton" name="selFrames">
     <property name="text">
      <string>Select Frames Folder</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="selExtractFolder">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Select Extraction Folder</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QPushButton" name="saveRSV">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Save RSV File</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="selRSV">
     <property name="text">
      <string>Select RSV File</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>108</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>selRSV</tabstop>
  <tabstop>selExtractFolder</tabstop>
  <tabstop>selFrames</tabstop>
  <tabstop>saveRSV</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
